src/staging/src/styles/404.scss

```diff
@@ -1,17 +1,20 @@
 @import "./global.scss";
 
-$header-height-404: 100px;
-$footer-height-404: 413px;
+$header-height-404: 220px;
+$footer-height-404: 390px;
```

Comment:
Ugh, I don't like this, but we didn't build it. I'd rather use flexbox properly than have to do this silly calculation. But this is ok for now.

Reply:
Yes, this calculation looks hacky and it's not a sustainable way. I deleted it because it will unlikely be less than 100vh. Minimum contents (header, content, and footer) are more than 100vh on most of the devices.